SafeguardSessions/README.md

```diff
@@ -150,6 +156,25 @@ If a section does not have related content, it can be left out.
 
 Modifications to the Report Template must be listed in the appropriate sections, since they are part of the release.
 
+## Refreshing your preloaded data
+
+### To refresh your preloaded data
+1. Open your previously saved .pbix file in Power BI Desktop.
+2. In Power BI Desktop, click **Home > Refresh**.
+
+If an error occures after refreshing your data, discard your changes to keep your data intact.
+
+## Upgrading the Power BI Connector
+
+If you are a new user who haven't used the custom connector yet, follow the steps of [Installation and usage].
+
+You can safely upgrade to a new version of the Power BI Connector if the major version number of the new Power BI Connector does not differ from the previous one. If it does, please view the Breaking changes section to be able to act upon it.
```

Comment:
Breaking changes section. Where can the use find this section? We should put a link to the Releases page. :slightly_smiling_face:

Reply:
Like this?

Reply:
No, I think it is misleading that we label a link "Breaking changes" and we point the user to the releases page. We should rather story-tell the user to go the releases page and see the breaking changes section of the appropriate release.

Reply:
Okay, I got it.

Reply:
Thanks! It's okay now! :slightly_smiling_face: